docs/ivys.js

```diff
@@ -72,22 +72,31 @@ var Product = function (state) {
 };
 
 
-Product.prototype.addTag = function () {
+Product.prototype.addTag = function (tagId, tagName) {
     // Add tag associated with product to product's state
     "use strict";
+    var self = this;
+    self.state.tags[tagId] = tagName;
 };
 
-Product.prototype.removeTag = function () {
+Product.prototype.removeTag = function (tagId) {
     // Remove tag associated with product from product's state
     "use strict";
+    var self = this;
+    delete self.state.tags[tagId];
 };
 
 // Product Collection Class
+// Example Maui-Collection, Milan Collection, etc
+
+// Should ProduceCollection have a State the way the Product class does? TODO
```

Comment:
It kind of is one. It doesn't really do anything else. I'd leave it as it for now.